### DISCUSSION TITLE
Help needed: Good dark mode settings

### DISCUSSION THREAD

**Discussion by AstralScribe** [qtile/qtile#4587] [General]
As I understand qtile doesn't provide global dark mode. Earlier I used i3 which used kde global dark mode by setting `XDG_CURRENT_DESKTOP=KDE`.  It didn't work in qtile for apps like chrome etc.

What happens is if I start my system in kde then logout and log back into qtile, the dark mode works perfectly. Also the command,

`qdbus org.freedesktop.portal.Desktop /org/freedesktop/portal/desktop org.freedesktop.portal.Settings.Read "org.freedesktop.appearance" "color-scheme"` 

Returns 1.

When starting qtile directly, the dark mode doesn't work and above command says it can't find `Settings.Read`. Also if I log back into kde, dark mode doesn't work there and above command fail with same reason.

**Comment by jwijenbergh** [comment]
> As I understand qtile doesn't provide global dark mode

Correct.

I do not recommend faking XDG_CURRENT_DESKTOP. Why not just set your gtk and qt themes?

**Comment by AstralScribe** [comment]
@jwijenbergh I tried that using qt5ct and lxappearance but it somehow borked some of my apps. Plus if I can find the exact service that setting the above value of colorscheme and add it to qtile autorun, it can solve the problem of no global dark mode. For gtk, I am using gsettings to set it to `prefer-dark` which seems to have solved my problem for gtk application.


I would also like to have some consistency across my two "desktops" kde and qtile, as it helps when other people have to use it.

**Reply by jwijenbergh** [reply]
Thanks, I have converted it to a discussion however due to it not being a qtile bug. Sorry but I don't really configure my gtk or qt themes, hopefully someone can chime in
